src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help.html

```diff
@@ -1,7 +1,7 @@
 <div>
     <p>
         Use this list to specify the MATLAB version for the build. The plugin prepends your specified MATLAB to the PATH environment variable and uses it to perform the build. <br>
-        The list includes the names of all MATLAB versions that have been registered as a tool in <u>Jenkins</u>. Select one of the tools, or select <b>Custom…</b> if you want to specify a different version. When you select <b>Custom…</b>, you must enter the full path to the desired MATLAB root folder in the <b>MATLAB root</b> box.
+        The list includes the names of all MATLAB versions that have been registered as a tool in Jenkins. Select one of the tools, or select <b>Custom…</b> if you want to specify a different version. When you select <b>Custom…</b>, you must enter the full path to the desired MATLAB root folder in the <b>MATLAB root</b> box.

```

Comment:
So apparently we shouldn't display ellipsis in the text. Please replace the two instances of Custom... with Custom.
This is only for the text. The UI label is fine as is.

Reply:
Done.